lib/features/auto-place/BpmnAutoPlace.js

```diff
@@ -1,18 +1,26 @@
-import { getNewShapePosition } from './BpmnAutoPlaceUtil';
+import { getNewShapePosition, getScrollOffset } from './BpmnAutoPlaceUtil';
 
 
 /**
  * BPMN auto-place behavior.
  *
  * @param {EventBus} eventBus
  */
-export default function AutoPlace(eventBus) {
+export default function AutoPlace(eventBus, canvas) {
   eventBus.on('autoPlace', function(context) {
     var shape = context.shape,
         source = context.source;
 
     return getNewShapePosition(source, shape);
   });
+
+
+  eventBus.on('autoPlace.end', function(event) {
+    var viewBox = canvas.viewbox();
+    var shape = event.shape;
```

Comment:
Just some personal preference: let's try to stick with our current style

```js
var viewBox = canvas.viewBox(),
    shape = event.shape;
```

also maps to other examples below